### ISSUE TITLE
SQLite does not support FULL JOIN

### ISSUE THREAD

**Issue by ianpartridge** [IBM-Swift/Swift-Kuery#19]
```
=======SELECT * FROM table1JoinOSX AS t1 FULL JOIN table2JoinOSX AS t2 USING (b)=======
Error in query:  RIGHT and FULL OUTER JOINs are not currently supported
```

**Comment by ianpartridge** [comment]
These can be emulated using LEFT JOIN and/or a UNION ALL clause.  See http://www.sqlitetutorial.net/sqlite-full-outer-join/

**Comment by irar2** [comment]
Addressed in #25
